addon/components/new-bourbon-select-field.js

```diff
@@ -102,7 +102,7 @@ export default Component.extend({
         'value',
         this.get('value')
       );
-      return this.get('internalContent').indexOf(valueHolder);
+      return this.get('prompt') ? this.get('internalContent').indexOf(valueHolder) : 0;
```

Comment:
this is for when no prompt is passed in and defaults to the first option in the dropdown.